tests/.env

```diff
@@ -16,3 +16,7 @@ AZURE_DEVOPS_PAT=
 AZURE_DEVOPS_PROJECT=
 AZURE_DEVOPS_BUILD_DEFINITON_ID=
 AZURE_DEVOPS_POOL_NAME=
+AZURE_KEYVAULT_URI=
+AZURE_AD_CLIENT_ID=
+AZURE_AD_CLIENT_SECRET=
+AZURE_AD_TENANT_ID=
```

Comment:
@tomkerkhove can we have these in GH secrets?

Reply:
Steps for E2E tests to work -

1. Provision a key vault using the subscription used for testing.
2. Register an application with Azure AD, and give the "read secrets" permission to it on the keyvault.
3. Add the keyvault uri, and AD credentials as GitHub secrets.

Reply:
By chance, is this possible to do via API as part of the test itself?

Reply:
I could try to do this by running shell commands from the test using the `az cli` but I don't think that's clean. And I don't know about the library to use for providing RBAC / access policy to app on the keyvault.

Reply:
I'll do it, it's OK.

Reply:
Please change to use these instead please:
- `AZURE_SP_ID`
- `AZURE_SP_KEY`
- `AZURE_SP_TENANT`

Reply:
https://github.com/kedacore/keda/pull/2756